src/components/Table/components/TableBody/index.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { ReactElement, ReactNode } from 'react';
 import clsx from 'clsx';
+import { RowData } from '../TableRow/types';
 import { Props } from './types';
+
+
```

Comment:
Suppose to be 2 new lines here even though it's still an import?

Reply:
I wasn't sure... That was mostly because the rest of the imports are other related components (`TableRow` in a `TableBody`). I'm open to changing it.

Reply:
I'd probably just put it between lines 2 and 3 like this

```
import React, { ReactElement, ReactNode } from 'react';
import clsx from 'clsx';
import TableRow from '../TableRow';
import { RowData } from '../TableRow/types';
import { Props } from './types';

Can't remember who, but I think imports usually go
```
npm packages
components
local files
```

I'm also open for separating like this, so I'll leave it up to you.